src/pages/iou/request/step/IOURequestStepDistance.js

```diff
@@ -211,7 +217,7 @@ function IOURequestStepDistance({
                 </View>
                 <View style={[styles.w100, styles.pt2]}>
                     {/* Show error message if there is route error or there are less than 2 routes and user has tried submitting, */}
-                    {((hasError && _.size(validatedWaypoints) < 2) || (_.keys(waypoints).length > 2 && _.size(validatedWaypoints) !== _.keys(waypoints).length) || hasRouteError) && (
+                    {hasError && (
```

Comment:
`hasError` is only used one time, so I think we should remove this variable

Reply:
@DylanDylann I removed `hasError`. Instead of this variable, I used `getError`.